pkg/sshutil/sshutil.go

```diff
@@ -155,6 +155,7 @@ func CommonArgs(useDotSSH bool) ([]string, error) {
 
 	args = append(args,
 		"-o", "StrictHostKeyChecking=no",
+		"-o", "UserKnownHostsFile=/dev/null",
```

Comment:
Do we now want to remove RemoveKnownHostEntries() in L101?

Reply:
> Do we now want to remove RemoveKnownHostEntries() in L101?

Yes, that function should not be needed anymore if we don't use a known hosts file.

@mook-as can you update the PR  to include removal of `RemoveKnownHostEntries()` (and the call location, of course)?

Reply:
Thanks for catching that! Fixed in 08d4bc3770410158242eb953c1eed38514e15015